### DISCUSSION TITLE
Welcome to ImpostorHQ Discussions!

### DISCUSSION THREAD

**Discussion by dimaguy** [dimaguy/ImpostorHQ#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other ImpostorHQ Users ~~(clearly not to just try this feature out)~~ . We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.
  * *Use this instead of issues for regular questions*

**Comment by le-show-de-didoub74** [comment]
- [ ] Create a Json where we can translate all Information in our language
- [ ] Modify `Impostor.Command.cfg` to add favorite language
- [ ] On the site add a button to change the language

**Reply by dimaguy** [reply]
I do not plan to add a translation system as it would be handicapped by plugin authors who cant be arsed to make translations for their own plugins
